class MethodCallError1 {
    void foo(Int value) {
        println(0);
    }
    
    void test() {
        foo();
    }
}
